app/models/journaled/client.rb

```diff
@@ -0,0 +1,56 @@
+class Journaled::Client
+  DEFAULT_REGION = 'us-east-1'.freeze
+
+  class << self
+    def generate
+      Aws::Kinesis::Client.new(kinesis_client_config)
```

Comment:
Should/could this class be called kinesis_client instead?

Reply:
Ya I think it might want to be! Good idea!

Reply:
Renamed!